src/components/anchor/code/templates/anchorTemplate.jsx

```diff
@@ -0,0 +1,16 @@
+var React = require('react');
+
+module.exports = function() {
+
+  if(!this.props.children) {
+    return null;
+  }
+
+  var dataAttributes = this.getDataAttributesFromProps();
+
+  return (
```

Comment:
I would argue anything that is above this line should be in the view, not the template. The pieces of logic above should not change if we decide to switch the template out.